### PR TITLE
[SW-1002] add back arrow icon to "back to main accounts" button

### PR DESCRIPTION
**Pull request by konradodo**

### CODE REVIEW

File: src/popup/components/ToggleMultisigButton.vue [hunk 1]
```diff
@@ -4,23 +4,32 @@
...
+
     <template v-else>
       <div
         v-if="hasPendingMultisigTransaction"
-        class="icon-wrapper"
+        class="has-pending"
       >
         <PendingIcon
-          class="animated-pending-icon"
+          v-if="hasPendingMultisigTransaction"
```
Comment (pstachel) [inline src/popup/components/ToggleMultisigButton.vue:22]: I don't see a point in doing that if clause here

File: src/popup/components/ToggleMultisigButton.vue [hunk 2]
```diff
@@ -89,22 +108,17 @@ export default defineComponent({
...
 
-    margin-right: 6px;
-
-    .tx-text {
-      @extend %face-sans-12-medium;
-
-      color: rgba($color-warning, 0.75);
-      line-height: 16px;
-    }
+    color: rgba($color-warning, 0.75);
```
Comment (pstachel) [inline src/popup/components/ToggleMultisigButton.vue:115]: So the square icon gets the opacity twice, right?

File: src/icons/arrow-back.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (CedrikNikita) [inline src/icons/arrow-back.svg:1]: But we already have a back button icon.

Comment (konradodo) [reply src/icons/arrow-back.svg]: good point, i found and replaced it

**Review by pstachel — APPROVED** [review]

File: src/popup/components/ToggleMultisigButton.vue [hunk 4]
```diff
@@ -4,23 +4,31 @@
...
+
     <template v-else>
       <div
         v-if="hasPendingMultisigTransaction"
-        class="icon-wrapper"
+        class="has-pending"
       >
         <PendingIcon
-          class="animated-pending-icon"
+          class="icon"
```
Comment (peronczyk) [inline src/popup/components/ToggleMultisigButton.vue:22]: I think it's a bad change to shorten the class name. Those two icons used in this component has different styles. Maybe you can share the common part this way:

```
.icon-foo,
.icon-bar {
  // Common styles
}
```

File: src/popup/components/ToggleMultisigButton.vue [hunk 5]
```diff
@@ -4,23 +4,31 @@
...
-        <span class="tx-text">{{ $t('common.tx') }}</span>
+        <span>
+          {{ $t('common.tx') }}
+        </span>
       </div>
-      <span class="text">{{ $t('multisig.showMultisigVaults') }}</span>
+
+      <span class="text">
+        {{ $t('multisig.showMultisigVaults') }}
+      </span>
```
Comment (peronczyk) [inline src/popup/components/ToggleMultisigButton.vue:31]: I don't see a reason for this change.

File: src/popup/components/ToggleMultisigButton.vue [hunk 6]
```diff
@@ -4,23 +4,31 @@
...
+        class="has-pending"
       >
         <PendingIcon
-          class="animated-pending-icon"
+          class="icon"
         />
-        <span class="tx-text">{{ $t('common.tx') }}</span>
+        <span>
+          {{ $t('common.tx') }}
+        </span>
```
Comment (peronczyk) [inline src/popup/components/ToggleMultisigButton.vue:26]: I don't see a reason for this change.

**Review by peronczyk — CHANGES_REQUESTED** [review]

File: src/popup/components/ToggleMultisigButton.vue (hunk 5 above)
Comment (konradodo) [reply src/popup/components/ToggleMultisigButton.vue:31]: for increased readability

File: src/popup/components/ToggleMultisigButton.vue (hunk 6 above)
Comment (konradodo) [reply src/popup/components/ToggleMultisigButton.vue:26]: for increased readability

File: src/popup/components/ToggleMultisigButton.vue [hunk 7]
```diff
@@ -4,23 +4,27 @@
...
         v-if="hasPendingMultisigTransaction"
-        class="icon-wrapper"
+        class="has-pending"
       >
         <PendingIcon
-          class="animated-pending-icon"
+          class="icon-pending"
         />
-        <span class="tx-text">{{ $t('common.tx') }}</span>
+        <span> {{ $t('common.tx') }} </span>
```
Comment (peronczyk) [inline src/popup/components/ToggleMultisigButton.vue:24]: Please remove the spaces around the `{{ ... }}`.

File: src/popup/components/ToggleMultisigButton.vue [hunk 8]
```diff
@@ -4,23 +4,27 @@
...
       </div>
-      <span class="text">{{ $t('multisig.showMultisigVaults') }}</span>
+
+      <span class="text"> {{ $t('multisig.showMultisigVaults') }} </span>
```
Comment (peronczyk) [inline src/popup/components/ToggleMultisigButton.vue:27]: Please remove the spaces around the `{{ ... }}`.

**Review by peronczyk — APPROVED** [review]

**Review by CedrikNikita — CHANGES_REQUESTED** [review]
Last commit is an obvious mistake.

**Comment by peronczyk** [comment]
Yeah, last commit should be removed.

**Pull request merged**